LanternExtractor/EQ/Wld/Exporters/GltfWriter.cs

```diff
@@ -291,6 +292,37 @@ public void GenerateGltfMaterials(IEnumerable<MaterialList> materialLists, strin
             }
         }
 
+        public void AddRegionData(Mesh mesh, DataTypes.BspNode frag)
+        {
+            var meshBuilder = new MeshBuilder<VertexPositionNormal>(mesh.Name);
+            var materialBuilder = new MaterialBuilder("");
+            var meshHelper = new WldMeshHelper(mesh, _separateTwoFacedTriangles);
+            var polygonCount = mesh.MaterialGroups.Aggregate(0, (acc, val) => acc + val.PolygonCount);
```

Comment:
TIL about LINQ Aggregate method. Though I wonder if it's more readable to do:
`mesh.MaterialGroups.Sum(m => m.PolygonCount)`
Assuming I'm not understanding your intent.

Reply:
TIL about Sum! That's a nice replacement. Changed